### PR TITLE
Added: New main dashboard

### PR DESCRIPTION
**Pull request by NinjaLabs-Dev**
Only layout and route checking for the moment, need to look at componenting the nav and setting up other pages.

### CODE REVIEW

**Comment by davwheat** [comment]
Does the route checking work? We're using hashes, not paths, for routing, so validating against pathname isn't something I've checked.

**Comment by NinjaLabs-Dev** [comment]
From my testing worked fine though feel free to send over another way

**Review by GewoonJaap — APPROVED** [review]

**Comment by GewoonJaap** [comment]
Merge?

**Comment by davwheat** [comment]
Just testing locally

**Comment by davwheat** [comment]
https://u.davwheat.dev/rPB3J2.png

Looks like the titlebar gets broken with these PR because of interference with the AppBar.

**Comment by NinjaLabs-Dev** [comment]
I did have an issue with this. It's due to the nav having z-index 1101 which I do try and override. I will have another look later

File: src/pages/LiveryManager.js [hunk 1]
```diff
@@ -1,12 +1,207 @@
...
+    setOpen(false);
+  };
+
   return (
-    <div>
-      <Typography variant="h4" gutterBottom>
-        Livery manager
-      </Typography>
+    <div className={styles.root}>
+      <AppBar
```
Comment (davwheat) [inline src/pages/LiveryManager.js:119]: The app bar is on top of the electron title bar. Fix coming.

File: src/pages/LiveryManager.js [hunk 2]
```diff
@@ -1,12 +1,207 @@
...
+  Drawer,
+  makeStyles,
+  IconButton,
+  useTheme,
+  ListItem,
+  ListItemIcon,
+  ListItemText,
+} from '@material-ui/core';
+
+import Devider from '@material-ui/core/Divider';
```
Comment (davwheat) [inline src/pages/LiveryManager.js:21]: Typo!

```suggestion
import Divider from '@material-ui/core/Divider';
```

**Review by davwheat — CHANGES_REQUESTED** [review]

**Comment by davwheat** [comment]
Changed base to the `add-manager` branch as this is all related to it

**Comment by davwheat** [comment]
Fixed the issues.

**Comment by NinjaLabs-Dev** [comment]
We moving over to top bar only then or keeping with drawer

**Comment by davwheat** [comment]
From Discord, the plan seems to be to move to a tab-base navigation system instead.

**Comment by NinjaLabs-Dev** [comment]
Oki, I'll move it over 😄

**Comment by GewoonJaap** [comment]
Gotta say, the drawer is also cool, :P

**Comment by davwheat** [comment]
@GewoonJaap I just think that a simple tab layout is simpler to see, and more apparent to users. In the future, we might need to consider a drawer instead, if we need to add more items to the manager.

**Comment by davwheat** [comment]
I'm going to merge into #18 to prevent simultaneous work towards the same objective on 2 branches.

**Pull request merged**